modules/blobs/endpoint.bal

```diff
@@ -98,11 +99,11 @@ public client class BlobClient {
     # + prefix - Optional. filters results to return only containers whose name begins with the specified prefix.
     # + return - If successful, returns ListBlobResult Else returns Error. 
     @display {label: "Get list of blobs"}
-    remote function listBlobs(@display {label: "Container name"} string containerName, 
-                              @display {label: "Max number of results"} int? maxResults = (), 
-                              @display {label: "nextMarker value from previous response"} string? marker = (), 
-                              @display {label: "Filter by prefix"} string? prefix = ()) 
-                              returns @tainted @display {label: "List of blobs"} ListBlobResult|error {
+    remote isolated function listBlobs(@display {label: "Container name"} string containerName, 
```

Comment:
This gets only metadata of blobs, not content in blobs. Javadoc should contain that information.

Reply:
Fixed